test/Parse/ConditionalCompilation/language_version_explicit.swift

```diff
@@ -1,6 +1,6 @@
-// RUN: %target-parse-verify-swift -swift-version 4
+// RUN: %target-parse-verify-swift -swift-version 3
 
-#if swift(>=4)
+#if swift(>=3)
```

Comment:
Yeah, based on Doug's preference for making an explicit whitelist (above) I can't really make a test that tests anything useful _yet_; you can only specify version 3, which is what the compiler is anyways. When there's more than one real version on the whiltelist, this test can be updated to be more interesting.